docs/configuration.md

```diff
@@ -37,12 +43,96 @@ If you are running these applications without Docker, just make sure that any en
-DEV_MODE should be `true`.
+**API_DEV_HOSTNAME** value to your development hostname:port, e.g. `myhost:8000` or `api.testserver.net`.
+**DEV_MODE** should be `true`.
+
+If you are deploying to a custom domain (not pol.is) than you need to update both the **API_PROD_HOSTNAME** and
+**DOMAIN_OVERRIDE** values to your custom hostname (omitting http(s):// protocol).
+**DEV_MODE** should be `false`.
+
+### General Settings
+
+- **ADMIN_UIDS** an array of user UUIDs for site admins. These users will have moderator capabilities on all conversations hosted on the site.
+- **EMAIL_TRANSPORT_TYPES** comma-separated list of email services to use (see [Email Transports](#email-transports) below)
+- **GIT_HASH** Set programatically using `git rev-parse HEAD` (e.g. in Makefile) to tag docker container versions and other release assets. Can be left blank.
+- **MATH_ENV** Set to prod (default), preprod, or dev. In cases where a single math service is used for multiple environments, this value is used by the API service to request the correct data.
+- **SERVER_ENV_FILE** The name of an environment file to be passed into the API Server container by docker compose. Defaults to ".env" if left blank. Used especially for building a "test" version of the project for end-to-end testing.
+- **SERVER_LOG_LEVEL** Used by Winston.js in the API server to determine how much logging to output. Reasonable values are "debug", "info", and "error". Defaults to "info".
+
+### Database
+
+- **READ_ONLY_DATABASE_URL** (optional) Database replica for reads.
+- **POSTGRES_DB** database name (e.g. "polis-dev")
+- **POSTGRES_HOST** database host (e.g. postgres:5432 if using docker compose, localhost:5432 if using local db)
+- **POSTGRES_PASSWORD** database password
+- **POSTGRES_PORT** typically 5432
+- **POSTGRES_USER** typically "postgres". Any username will be used by the docker container to create a db user.
+- **DATABASE_URL** should be the combination of above values, `postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}/${POSTGRES_DB}`
+
+### Docker Concerns
+
+- **TAG** used by **COMPOSE_PROJECT_NAME** below. Defaults to "dev".
+- **COMPOSE_PROJECT_NAME** Used by docker compose to label containers and volumes. Useful in development if you are (re)-building and deleting groups of docker assets.
+
+### Ports
+
+- **API_SERVER_PORT** typically 5000. Used internally within a docker network and/or behind a proxy.
+- **HTTP_PORT** typically 80. Port exposed by Nginx reverse proxy.
+- **HTTPS_PORT** typically 443. Port exposed by Nginx reverse proxy.
+- **STATIC_FILES_PORT** typically 8080. Used internally within a docker network and/or behind a proxy.
+- **STATIC_FILES_ADMIN_PORT** same as **STATIC_FILES_PORT** unless you are hosting client-admin separately from file-server. Useful in local development.
+- **STATIC_FILES_CLIENT_PORT** same as **STATIC_FILES_PORT** unless you are hosting client-participation separately from file-server. Useful in local development.
```

Comment:
This should be `STATIC_FILES_PARTICIPATION_PORT`, since "client" doesn't really disambiguate here.